### PR TITLE
Fix assignment list in config

### PR DESCRIPTION
**Pull request by kcranston**
This addresses the two problems from issue #161 :

* does not add assignment to README text in config
* will only add the same assignment once to the config

### CODE REVIEW

**Comment by kcranston** [comment]
The remaining Codacy complaint is the changelog format (i.e. we can ignore it).

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -1,93 +1,68 @@
...
-^^^^^^^^^^^^^
+=============
+
 To run all of the tests (from the root directory of the repo)::
 
     $ pytest
 
 To run only the subset of the tests in (for example) `singlefile.py`, use:
 
-    $ pytest abcclassroom/tests/single_file.py
```
Comment (lwasser) [inline CONTRIBUTING.rst]: @kcranston can we please put this bump version docs back into this file?

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -1,93 +1,68 @@
...
-1.0.0), a minor release (e.g., 0.1.0 to 0.2.0), or a patch (e.g., 0.1.0 to
-0.1.1), following the guidelines for semantic versioning: https://semver.org/.
-
-
-Bumpversion updates the version number throughout the
-package, and generates a git commit along with an associated git tag for the
-new version.
-For more on bumpversion, see: https://github.com/peritus/bumpversion
-
-To deploy EarthPy, push the commit and the version tags::
```
Comment (lwasser) [inline CONTRIBUTING.rst]: this should be abc-classroom

**Comment by lwasser** [comment]
this can be merged after the small text fixes to docs!

**Pull request merged**